datalab/mlalpha/commands/_mlalpha.py

```diff
@@ -997,10 +1012,17 @@ def _feature_slice_view(args, cell):
 browser.weightedExamplesColumn = 'totalWeightedExamples';
 browser.calibrationPlotUriFn = function(s) { return '/' + s; }
 </script>"""
-  with open(args['file']) as f:
-    data = map(json.loads, f)
-  if args['feature']:
-    data = [e for e in data if e['feature'].split(':')[0] == args['feature']]
+  if args['sql'] is not None:
+    item = datalab.utils.commands.get_notebook_item(args['sql'])
+    item, _ = datalab.data.SqlModule.get_sql_statement_with_environment(item, {})
+    query = datalab.bigquery.Query(item)
+    df = query.results().to_dataframe()
+    data = _get_lantern_format(df)
+  elif args['dataframe'] is not None:
+    item = datalab.utils.commands.get_notebook_item(args['sql'])
```

Comment:
Good catch. The tests for magics are mostly sample notebooks which will be run in tests and make sure no exception is thrown. The sample notebook is still under construction.

Fixed.